limitlion/running_counter.py

```diff
@@ -106,20 +106,31 @@ def _get_name(self, name):
                 raise ValueError('Name not specified')
             return name
 
-    def _all_buckets(self, now):
+    def _all_buckets(self, recent_periods=None, now=None):
         """
         Get all buckets in running counter's window.
         """
```

Comment:
Maybe now add a docstring here like we do for `buckets`, since now we have more arguments and what they do is not completely obvious from their names?

Reply:
Definitely 👍